docs/docusaurus/versioned_docs/version-1.5.X/orc8r/upgrade_1_5.md

```diff
@@ -66,7 +66,7 @@ This process prepares your cluster for Orc8r-NMS DB Unification.
 *Paste in shell prompt:*
 
 ```bash
-wget https://raw.githubusercontent.com/magma/magma/master/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
+wget https://raw.githubusercontent.com/magma/magma/master/nms/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
```

Comment:
And leave behind broken links?

Reply:
I think the point here is that people should be able to run the migration script and we should probably point to the latest version in case we want to fix something.